edb/pgsql/codegen.py

```diff
@@ -156,25 +165,30 @@ def ctes_to_source(cls, ctes: List[pgast.CommonTableExpr]) -> str:
         return ''.join(generator.result)
 
     @classmethod
-    def to_source_with_translation(
+    def to_source_ex(
```

Comment:
what's `_ex`? Extended?

Reply:
Yeah. "extended" or "expert" (I'm not sure what the origin for `Ex` was in WinAPI). Suggestions for a better name welcome.